components/main_page/wikis/arenafps/main_page_layout_data.lua

```diff
@@ -24,7 +24,7 @@ local CONTENT = {
 		body = '{{Transfer List|limit=15}}\n<div style{{=}}"display:block; text-align:center; padding:0.5em;">\n' ..
 			'<div style{{=}}"display:inline; float:left; font-style:italic;">\'\'[[#Top|Back to top]]\'\'</div>\n' ..
 			'<div style{{=}}"display:inline; float:right;" class="plainlinks smalledit">' ..
-			'&#91;[{{FULLURL:Player Transfers/{{CURRENTYEAR}}/1st_Quarter|action=edit}} edit]&#93;</div>\n' ..
+			'&#91;[{{FULLURL:Player Transfers/{{CURRENTYEAR}}/{{Current term}}|action=edit}} edit]&#93;</div>\n' ..
```

Comment:
I think Current term insert current year too, check that the year is not duplicated

Reply:
Ya, you're right, does also insert the year, did update that just now